corehq/apps/hqadmin/views/users.py

```diff
@@ -92,31 +92,21 @@ def post(self, request, *args, **kwargs):
             users = form.cleaned_data['users']
             is_superuser = 'is_superuser' in form.cleaned_data['privileges']
             is_staff = 'is_staff' in form.cleaned_data['privileges']
-
-            changed_fields = []
+            changed_field_logs = []
             for user in users:
                 # save user object only if needed and just once
                 if user.is_superuser is not is_superuser:
-                    update_log = {
-                        'field': 'is_superuser',
-                        'previous': user.is_superuser,
-                        'current': is_superuser,
-                    }
                     user.is_superuser = is_superuser
-                    changed_fields.append(update_log)
+                    changed_field_logs.append(_("{} to {}".format("is_superuser", is_superuser)))
 
                 if can_toggle_is_staff and user.is_staff is not is_staff:
-                    update_log = {
-                        'field': 'is_staff',
-                        'previous': user.is_staff,
-                        'current': is_staff,
-                    }
                     user.is_staff = is_staff
-                    changed_fields.append(update_log)
+                    changed_field_logs.append(_("{} to {}".format("is_staff", is_staff)))
 
-                if changed_fields:
+                if changed_field_logs:
                     user.save()
-                    log_model_change(self.request.user, user, fields_changed=changed_fields)
+                    log_message = _("Changed: ") + _(" and ").join(changed_field_logs)
```

Comment:
I believe you can keep it as simple as 
1. `changed_field_logs.append(f"is_superuser {is_superuser}")`
2. and change here to ",".join(changed_field_logs)

These are not user facing messages, so don't really need to be pretty.

Reply:
Done.